Hospital/src/patterns/RoomFactory.java

```diff
@@ -0,0 +1,17 @@
+package patterns;
+
+public class RoomFactory {
+	
+	public Room getRoom(String tipo){
+		
+		if (tipo.equals("cuarto comun"))
+			return new HabitacionComun();
+		if (tipo.equals("quirofano"))
```

Comment:
[Low] To avoid *NullPointerException* while comparing, it is a good practice to compare in this way:
```java
if ("quirofano".equals(tipo)) {}
```
In this case, if *tipo* is <code>null</null>, the the condition is *false* but it doesn't throw an error.